apps/workflows-dashboard/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:18.16-alpine3.17 as dev
+
+WORKDIR /app
+
+COPY . .
+
+RUN mv /app/.env.example /app/.env; rm -rf /app/node_modules; npm install --legacy-peer-deps ; npm run build
```

Comment:
Can you reach out Ilia to check why we need --legacy-peer-deps?

Reply:
@pratapalakshmi  We don't need `legacy-peer-deps`, base installation should work just fine.
From where it comes?

Reply:
I am getting error when i do `npm install` and that why i had to do `npm install --legacy-peer-deps`

Reply:
I think problem is that package.lock is missing since we using `pnpm`
I would suggest to copy monorepo in docker file including pnpm lock file,then cd to dashboard and install deps using `pnpm instal` (before that need to install pnpm in container)